examples/checkbox/checkbox.colors.tsx

```diff
@@ -0,0 +1,87 @@
+import { type CodeData } from '~/components/code-demo';
+import { Checkbox, Label } from '~/src';
+
+const code = `
+'use client';
+
+import { Checkbox, Label } from 'flowbite-react';
+
+function Component() {
+  return (
+    <div>
+      <div className="mb-4 flex items-center gap-2">
+        <Checkbox id="default-checkbox" />
+        <Label htmlFor="default-checkbox">Default checkbox</Label>
+      </div>
+      <div className="flex items-center gap-2">
+        <Checkbox id="checked-state-checkbox" checked />
+        <Label htmlFor="checked-state-checkbox">Checked state</Label>
+      </div>
+    </div>
+  );
+}
+`;
+
+const codeRSC = `
+import { Checkbox, Label } from 'flowbite-react';
+
+function Component() {
+  return (
+    <div>
+      <div className="mb-4 flex items-center gap-2">
+        <Checkbox id="default-checkbox" />
+        <Label htmlFor="default-checkbox">Default checkbox</Label>
+      </div>
+      <div className="flex items-center gap-2">
+        <Checkbox id="checked-state-checkbox" checked />
+        <Label htmlFor="checked-state-checkbox">Checked state</Label>
+      </div>
+    </div>
+  );
+}
+`;
+
+function Component() {
+  return (
+    <div className="flex items-center gap-2">
+      <div className="flex items-center gap-2">
+        <Checkbox id="red-checkbox" color="red" checked />
+        <Label htmlFor="red-checkbox">Red</Label>
+      </div>
+      <div className="flex items-center gap-2">
+        <Checkbox id="green-checkbox" color="green" checked />
+        <Label htmlFor="green-checkbo">Green</Label>
+      </div>
+      <div className="flex items-center gap-2">
+        <Checkbox id="purple-checkbox" color="purple" checked />
+        <Label htmlFor="purple-checkbo">Purple</Label>
+      </div>
+      <div className="flex items-center gap-2">
+        <Checkbox id="teal-checkbox" color="teal" checked />
+        <Label htmlFor="teal-checkbo">Teal</Label>
+      </div>
+      <div className="flex items-center gap-2">
+        <Checkbox id="yellow-checkbox" color="yellow" checked />
+        <Label htmlFor="yellow-checkbo">Yellow</Label>
+      </div>
```

Comment:
I think orange is missing

Reply:
orange is not in FlowBiteColors ?

Reply:
oh that's true, but why isn't it 🤔

Reply:
There are all the flowbite colors:
https://flowbite.com/docs/customize/colors/

However, it extends the TailwindCSS... so it means that...
https://tailwindcss.com/docs/customizing-colors

☝️ these colors (if now overwriten) are also available.

Reply:
I see, gotcha! 🤝

Reply:
BTW, just one side note here if someone wants to create an issue for this (not asking to include on this PR), we should work to provide all colors to all the components, and improve the color interface `FlowbiteColors`, `TailwindColors`, `FlowbiteStateColors` (default, active, enable, disable)... I don't know. But, we need to create some standard interface that needs to be available to all the components. Needs some PoC/Proposal/Investigation, maybe not all components need all the colors, and in this case, we may need to find a way to inform it in the docs.

But as I said, I don't expect this to be fixed on this PR... But we need a very well-described issue for this. (ChatGPT?)

Reply:
lets get to digging/brainstorming then